deploy/addons/coredns/coreDNS-svc.yaml

```diff
@@ -1,16 +1,16 @@
 apiVersion: v1
 kind: Service
 metadata:
-  name: coredns
+  name: kube-dns
   namespace: kube-system
   labels:
-    k8s-app: coredns
-    kubernetes.io/name: "CoreDNS"
+    k8s-app: kube-dns
     addonmanager.kubernetes.io/mode: Reconcile
+    kubernetes.io/name: "CoreDNS"
 spec:
   selector:
-    k8s-app: coredns
-  clusterIP: 10.0.0.10
+    k8s-app: kube-dns
+  clusterIP: 10.96.0.10
```

Comment:
This needs to match kubelet, are you sure we want to change it?

Reply:
It's changed for kube-dns as well. Keeping in sync.

Reply:
This was changed in https://github.com/kubernetes/minikube/pull/2077

Sorry for missing it in this addon!